crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -219,7 +219,7 @@ impl RoomEventCache {
             .state
             .read()
             .await
-            .find_event(event_id)
+            .find_event_by_id(event_id)
```

Comment:
As discussed, maybe we can keep the same name, as the `_by_id` suffix doesn't give us extra information?